ui/src/services/group-item-service/group-item-service.ts

```
@@ -60,8 +69,8 @@ export class GroupItemService {
     return this.devicesQuery.result
   }
 
-  get currentGroup(): GroupListResponseGroupsItem | undefined {
-    return this.groupSettingsService.groupsQueryResult.data?.find((item) => item.id === this.currentGroupId)
+  get currentGroup(): GroupListResponseGroupsItem {
+    return this.groupSettingsService.groupsQueryResult.data?.find((item) => item.id === this.currentGroupId) || {}
```

Why {} instead of undefined or null?

---

This behavior (lack of a group) is not expected, because this is a service for a specific group that was previously received. (see ```group-list.tsx```). If during the user's setup this group disappears (for example, it is deleted by someone else), then this group will also disappear for the user.

I checked the entire project - we do not expect any errors from this change.

---

Maybe throw an error then?